README.md

```diff
@@ -14,3 +14,11 @@ Coming soon:
 - tests for our code!
 - feedback box for more questions and other suggestions
 - moderated discussion page(s) 
+
+
+## Installation instructions (development)
+
```

Comment:
can you add an instruction to install the right django version? that was the one bump I encountered when trying to get it run locally